src/js/modules/components/modal.js

```diff
@@ -0,0 +1,49 @@
+import { View } from 'backbone';
+import modalTemplate from './modal.hbs';
+
+const HIDDEN_CLASS = 'hidden';
+const CONTENT_CLASS = 'plantingjs-modal-content';
+
+export default View.extend({
+  events: {
+    'click .plantingjs-modal-background': 'hide',
+    'click .plantingjs-modal-close-btn': 'hide',
+  },
+
+  constructor({ childView, ...opts }) {
+    this.childView = childView;
+    View.call(this, opts);
+  },
+
+  initialize() {
+    this.render();
+    this.$el
+      .toggleClass(HIDDEN_CLASS, false)
+      .find(`.${CONTENT_CLASS}`)
+        .html(this.childView.$el);
```

Comment:
What's the reason of incrising intendation here? Below You also have chain of methods and You're not incrising intendation.